### ISSUE TITLE
After loading Local LLM, console displays 'Constant' object has no attribute 'kind'

### ISSUE THREAD

**Issue by 641217688tly** [ise-uiuc/TitanFuzz#4]
(titanfuzz) [tly@localhost TitanFuzz]$ bash scripts/demo_run_torch.sh false
Warning: running in a non-docker environment!
Current directory:  /home/tly/Fuzzing/TitanFuzz/TitanFuzz
Results will be dumped to:  /home/tly/Fuzzing/TitanFuzz/TitanFuzz/Results
[0 / 2] torch.nn.RReLU
output.returncode:  0
stdout>  Current directory:  /home/tly/Fuzzing/TitanFuzz/TitanFuzz
Results will be dumped to:  /home/tly/Fuzzing/TitanFuzz/TitanFuzz/Results
> api:  torch.nn.RReLU
Initializing a SpanLM based model: facebook/incoder-1B ...
pretrained: facebook/incoder-1B
Model loaded from: /home/tly/Fuzzing/TitanFuzz/TitanFuzz/LLM/incoder-1B
Model configuration: XGLMConfig {
  "_name_or_path": "/home/tly/Fuzzing/TitanFuzz/TitanFuzz/LLM/incoder-1B",
  "activation_dropout": 0.0,
  "activation_function": "gelu",
  "architectures": [
    "XGLMForCausalLM"
  ],
  "attention_dropout": 0.1,
  "attention_heads": 32,
  "bos_token_id": 0,
  "d_model": 2048,
  "decoder_start_token_id": 2,
  "dropout": 0.1,
  "eos_token_id": 2,
  "ffn_dim": 8192,
  "init_std": 0.02,
  "layerdrop": 0.0,
  "max_position_embeddings": 2048,
  "model_type": "xglm",
  "num_layers": 24,
  "pad_token_id": 1,
  "scale_embedding": true,
  "torch_dtype": "float32",
  "transformers_version": "4.39.3",
  "use_cache": true,
  "vocab_size": 50518
}

Max length: 2048
Batch size: 30
Model loading time: 45.33568048477173
**'Constant' object has no attribute 'kind'**

[1 / 2] torch.nn.Conv2d
output.returncode:  0
stdout>  Current directory:  /home/tly/Fuzzing/TitanFuzz/TitanFuzz
Results will be dumped to:  /home/tly/Fuzzing/TitanFuzz/TitanFuzz/Results
> api:  torch.nn.Conv2d
Initializing a SpanLM based model: facebook/incoder-1B ...
pretrained: facebook/incoder-1B
Model loaded from: /home/tly/Fuzzing/TitanFuzz/TitanFuzz/LLM/incoder-1B
Model configuration: XGLMConfig {
  "_name_or_path": "/home/tly/Fuzzing/TitanFuzz/TitanFuzz/LLM/incoder-1B",
  "activation_dropout": 0.0,
  "activation_function": "gelu",
  "architectures": [
    "XGLMForCausalLM"
  ],
  "attention_dropout": 0.1,
  "attention_heads": 32,
  "bos_token_id": 0,
  "d_model": 2048,
  "decoder_start_token_id": 2,
  "dropout": 0.1,
  "eos_token_id": 2,
  "ffn_dim": 8192,
  "init_std": 0.02,
  "layerdrop": 0.0,
  "max_position_embeddings": 2048,
  "model_type": "xglm",
  "num_layers": 24,
  "pad_token_id": 1,
  "scale_embedding": true,
  "torch_dtype": "float32",
  "transformers_version": "4.39.3",
  "use_cache": true,
  "vocab_size": 50518
}

Max length: 2048
Batch size: 30
Model loading time: 41.063090324401855
**'Constant' object has no attribute 'kind'**

**Comment by panpanxiduoduo** [comment]
me too。how to solve
